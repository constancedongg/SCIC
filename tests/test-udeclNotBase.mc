|'{mm} = 10.00 '{cm}|;

int func main(){
    float '{m} a = 1.23;
    float '{cm} b = 45.67;
    float '{mm} p = a;
    printf(p);
    p = b;
    printf(p);
    return 0;
}