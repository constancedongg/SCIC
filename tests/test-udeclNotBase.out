1230
456.7